modules/exploits/windows/local/lenovo_systemupdate.rb

```diff
@@ -0,0 +1,193 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class Metasploit3 < Msf::Exploit::Local
+  Rank = ExcellentRanking
```

Comment:
@jvazquez-r7 I know you're the ranking master here :) Should this be Normal or Excellent?